### DISCUSSION TITLE
Unable to start LinuxGSM

### DISCUSSION THREAD

**Discussion by Shade-cs** [GameServerManagers/LinuxGSM#3537] [General]
* Gameserver: Minecraft
* Ubuntu 18.04.5 LTS
* No logs
* The output of ./gameserver postdetails https://termbin.com/q59y
* Hardware: 12Cores 16gb ram and enough free storage.
* Tried restarting server, didn't help.
* Currently got 3 other servers running, but ofc on different ports. (CS:S, CS:GO & Unturned)

```
~$ ./mcserver start
[ FAIL ] Starting mcserver: Unable to start LinuxGSM
```

My 3 other LGSM gameservers starts fine.

**Comment by Shade-cs** [comment]
Problem fixed by 2 helpfull folks on discord. (PlayMTL#8338 & 🐫 DefenderOfHyrule 🐫#8758)
Issue: Java was too old. (Running Java 11 with mc 1.7+ doesnt work, you need to update to java 16 and enable by default )

**Reply by ghost** [reply]
Could you please tell what did you do to update to version 16? I'm on a debian server and I'm not able to do it yet
